src/rules/media-feature-name-blacklist/__tests__/index.js

```diff
@@ -0,0 +1,60 @@
+﻿import {
+  messages,
+  ruleName,
+  } from ".."
+import rules from "../../../rules"
+import { testRule } from "../../../testUtils"
+
+const rule = rules[ruleName]
+
+testRule(rule, {
+  ruleName,
+  config: [[
+    "max-width",
+    "--max-width",
+    "width",
+    "/^my-/",
+  ]],
+
+  accept: [ {
+    code: "@media (min-width: 50em) { padding: 1rem; }",
+  }, {
+    code: "@media (min-width: 50em) { max-width: 50em; }",
+  }, {
+    code: "@media (--max-width: 50em) { padding: 1rem; }",
```

Comment:
I'm not sure this is valid.

*Custom properties* are only found in declaration blocks. *Custom media queries* have [no value](https://www.w3.org/TR/2016/WD-mediaqueries-4-20160126/#custom-mq) e.g.

```css
@media (--wide-viewport) {}
```

I see you're already correctly ignoring custom media queries, and so I'd remove `--max-width` from the config